src/Form/Formbot.js

```diff
@@ -97,24 +96,21 @@ export default class Formbot extends React.Component {
       cb
     );
 
-  updateField(field, updates = {}) {
-    return new Promise(resolve => {
-      const fieldState = this.state.fields[field] || {};
-
+  updateField = (field, updates = {}) =>
+    new Promise(resolve => {
       this.setState(
-        {
+        state => ({
           fields: {
-            ...this.state.fields,
+            ...state.fields,
             [field]: {
-              ...fieldState,
+              ...state.fields[field],
```

Comment:
Does this no longer require the default `{}` value?

Reply:
@cehsu 
```
{ ...undefined, ...null } == {}
```